networks.json

```diff
@@ -32,8 +32,8 @@
       "transactionHash": "0x4e20d94594f7ae340dadb18a9b7fa10e80f0a8dae2c72c27442cb015b9b205cf"
     },
     "4": {
-      "address": "0x7B0F8dEE044af57eb9D57Fb16A8738cd84aB1f9E",
-      "transactionHash": "0xd1030e85508042a6762598935751d38c4acd537401e2f2980c8fba5261022691"
+      "address": "0x0AD02243a47eB21Bb0EBd9986d99c0605039100e",
+      "transactionHash": "0xfc3066fa7632f9ce8a0f258ce3d5886d9f2083c47ca27c17a5a0db466f419a53"
```

Comment:
It would be nice to also have this contract verified on Etherscan, it's not done automatically because it's deployed in the contract constructor. Maybe we should add an issue and work on it eventually?

Reply:
https://github.com/gnosis/gp-v2-contracts/issues/322